### ISSUE TITLE
Compatibility with Pixel Fold

### ISSUE THREAD

**Issue by notmydevice** [KieronQuinn/PixelLauncherMods#126]
Unfortunately, this gem is not presently compatible with Pixel Fold.

Things work as expected until the device is unfolded, at which point Pixel Launcher crashes.

Is there a plan to support this Pixel device?

**Comment by notmydevice** [comment]
UPDATE: this issue has been traced to the widget replacement feature.

When widget replacement is active, unfolding the device causes Pixel Launcher to crash. This happens with every replacement widget, even the "blank" widget that ships with Pixel Launcher Mods.

Hopefully this can be addressed. Pixel Launcher Mods provides an enormous amount of useful functionality. But the ability to remove the giant, hideous Google billboard from the user's home screen is its greatest superpower.

Thanks @KieronQuinn for all you do.

**Comment by KieronQuinn** [comment]
That's interesting because my assumption was it was the icons. Can you get a log?

**Comment by KieronQuinn** [comment]
You shouldn't need to if you filter the log by `AndroidRuntime` (no sensitive data will be included), but if you want to anyway there's an email on https://kieronquinn.co.uk/

**Comment by notmydevice** [comment]
Yep, just found that, my edit collided with your response. Will get this to you asap.

**Comment by notmydevice** [comment]
Hey @KieronQuinn, you should have the log. Hope you don't mind that I emailed it. Very happy to assist in any way I can, let me know what I can do.

**Comment by fsmv** [comment]
Hi any news on this? Did you get the logs? If not I can probably pull some from my phone.

**Comment by notmydevice** [comment]
@KieronQuinn, any chance this issue is magically resolved on the just-released Pixel 9 Pro Fold? Could certainly motivate one to upgrade.
